### DISCUSSION TITLE
Make Value Boxes possible outside of format: dashboard

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#8413] [Feature Requests]
### Description

I really like the look of the [Value Boxes added in Version 1.4](https://quarto.org/docs/dashboards/data-display.html#value-boxes)

Right now it is not possible to add them in the following way when the format is not dashboard. 

```{python}
#| content: valuebox
#| title: "Articles per day"
#| icon: pencil
#| color: primary
dict(
  value = articles
)
```

I know there is a way through using [R and the bslib-package](https://rstudio.github.io/bslib/articles/value-boxes/) but I don't have access to R. 

Is there an easy way to achieve this or will it be possible in 1.5?

**Comment by mcanouil** [comment]
The easiest way to get the answer is usually to try.
Note that providing a reproducible example is always better when asking something (see <https://quarto.org/bug-reports.html>)

Below a 100 % reproducible example showing that value box is a dashboard feature only (also why it is only described in the dashboard part of the documentation, even though it's a work in progress towards the release).

This being said, It might be possible to allow dashboard features in "regular" html for not for 1.4.

CC @dragonstyle is it something doable in 1.5?

````qmd
---
title: "Quarto Playground"
format: html
---

This is a playground for Quarto.

```{python}
#| content: valuebox
#| title: "Articles per day"
#| icon: pencil
#| color: primary
dict(
  value = "a"
)
```
````

**Reply by dragonstyle** [reply]
I think if we have time this is something that could be supported, though I think it will require a bit of investment to separate the processing of value boxes (to generalize this in LUA, likely adding a new custom node) and a small / easy task to inject the proper dependencies.

**Reply by mcanouil** [reply]
Basically, something similar to callouts if I am not mistaken.

**Reply by mcanouil** [reply]
I opened an issue to track this:
- #8475
